mods/content/government/away_sites/icarus/icarus.dm

```diff
@@ -171,7 +171,6 @@
 	name = "expeditionary uniform"
 	desc = "An older model of the utility uniform of the SCG Expeditionary Corps. It has a patch on the left sleeve signifying the wearer served on the SEV Icarus."
 	icon_state = "blackutility_crew"
```

Comment:
is that supposed to be still set?

Reply:
No, but I have it fixed in another commit (as discussed) and would prefer not to mess with these commits at this point.
Once they're all merged I'll use the debug verb added in a more recent PR to do a fix pass.